### PR TITLE
Get subtotal() for total() without any format

### PR DESCRIPTION
**Pull request by ebisbe**

### CODE REVIEW

File: src/Darryldecode/Cart/ItemCollection.php [hunk 1]
```diff
@@ -115,7 +115,11 @@ public function getPriceWithConditions()
 
             return number_format($newPrice, $this->decimals, $this->dec_point, $this->thousands_sep);
```
Comment (it-can) [inline src/Darryldecode/Cart/ItemCollection.php]: This line also

File: src/Darryldecode/Cart/ItemCollection.php [hunk 2]
```diff
@@ -125,6 +129,6 @@ public function getPriceWithConditions()
      */
     public function getPriceSumWithConditions()
```
Comment (it-can) [inline src/Darryldecode/Cart/ItemCollection.php]: please add ```$formatted``` also here...

**Comment by ebisbe** [comment]
wait please. I'm working on it.

**Comment by ebisbe** [comment]
@it-can Please check if I missed something.

**Comment by it-can** [comment]
Looks good, except typo with $formated instead of $formatted

**Comment by it-can** [comment]
This is a nice solution btw

**Comment by ebisbe** [comment]
@it-can Typo fixed. I added some tests with 3 decimals, decimal separator comma and thousands separator dot in order to test something completely different.

**Comment by it-can** [comment]
Very nice

**Comment by it-can** [comment]
@darryldecode please merge this PR

**Comment by darryldecode** [comment]
👍

**Pull request merged**